AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/edit-work-hours/vho-search/vho-search.component.ts

```diff
@@ -47,14 +47,15 @@ export class VhoSearchComponent implements OnInit {
                         result = await this.service.getNonWorkAvailabilityForVho(this.username.value);
                         break;
                 }
-
-                if (result) {
-                    this.vhoSearchEmitter.emit(result);
+                var myItems = result.slice(0, 20);
```

Comment:
Check the sonar cloud analysis results, they have reported the following code smell:

`Unexpected var, use let or const instead`

I'd suggest also using an alternative name and a private property or constant to make the intention clearer. Something like:

`private maxPageSize = 20;`

`const filteredResult = result.slice(0, maxPageSize);`